src/utils/quote_server_client.ts

```diff
@@ -28,7 +28,11 @@ export class QuoteServerClient {
             const response = await this._axiosInstance.post(`${makerUri}/submit`, payload, {
                 timeout: ONE_SECOND_MS * 2,
             });
-            schemaValidator.validate(response.data, schemas.submitReceiptSchema);
+            const validator = schemaValidator.validate(response.data, schemas.submitReceiptSchema);
+            if (validator.errors.length > 0) {
```

Comment:
@phil-ociraptor `validate` does not throw an error by default, so I just force it to throw an error if errors are present.

Reply:
hmmm.... I wonder how some of my tests were passing, lol, but all good!